.github/workflows/publish.yml

```
@@ -17,42 +16,55 @@ permissions:
   contents: read
 
 jobs:
-  # calls our general CI workflow (tests, build docs, etc.)
+  # calls our general CI workflows (tests, coverage, profile, etc.)
   tests:
     uses: ./.github/workflows/CI.yml
     # needed for the coverage action
```

Once this is merged, this will need pinning to a SHA.

---

shouldn't this one also change to: 

```suggestion
    uses: pydata/pydata-sphinx-theme/.github/workflows/CI.yml
    # needed for the coverage action
```